setup/linux/build_deb.sh

```diff
@@ -24,7 +24,7 @@ PACKAGE_NAME="mmex-$MMEX_VERSION-$ARCHITECTURE"
 cd ../..
 MMEX_DIR=`pwd`
 
-./bootstrap
+./bootstrap.sh
```

Comment:
cool

2015-05-14 17:23 GMT+03:00 David Jones <notifications@github.com>:

> In setup/linux/build_deb.sh
> <https://github.com/moneymanagerex/moneymanagerex/pull/474#discussion_r30324507>
> :
>
> > @@ -24,7 +24,7 @@ PACKAGE_NAME="mmex-$MMEX_VERSION-$ARCHITECTURE"
> >  cd ../..
> >  MMEX_DIR=`pwd`
> >
> > -./bootstrap
> > +./bootstrap.sh
>
> I've renamed bootstrap to sh since it is a sh file and the .gitattributes
> file will cover it then without adding a new rule
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/moneymanagerex/moneymanagerex/pull/474/files#r30324507>
> .
>